### PR TITLE
RA-469: Add load image from icon in Misc

### PR DESCRIPTION
**Pull request by aameijide**

### CODE REVIEW

File: VAS.UI.Helpers.Gtk2/Misc.cs [hunk 1]
```diff
@@ -235,11 +235,29 @@ public static Gdk.Pixbuf LoadIcon (string name, int size, IconLookupFlags flags
...
+		/// <returns>The icon as a Gdk.Image or missing image icon if not found.</returns>
+		/// <param name="name">Icon Name.</param>
+		/// <param name="width">Icon Width as a Gtk.IconSize.</param>
+		/// <param name="width">Icon Height as a Gtk.IconSize.</param>
+		/// <param name="flags">Lookup Flags like ForceSVG.</param>
+		public static Gtk.Image LoadImageIcon (string name, int width, int height, IconLookupFlags flags = IconLookupFlags.ForceSvg)
+		{
+			Gtk.Image image = new Gtk.Image ();
+			image.Pixbuf = Misc.LoadIcon (name, width);
+			image.WidthRequest = width;
```
Comment (vmartos) [inline VAS.UI.Helpers.Gtk2/Misc.cs]: I don't think having a widthRequest and HeightRequest here is a good idea, by default all icon images has to be square sized: see https://fluendo.atlassian.net/wiki/display/LON/Graphic+resources%3A+icons+and+images#suk=ff808081520a3deb01521c0225e10002
So we need to only call Helpers.Misc.LoadIcon ("imagepath", StyleConf.Size)

File: VAS.UI.Helpers.Gtk2/Misc.cs [hunk 2]
```diff
@@ -240,19 +240,18 @@ public static Gdk.Pixbuf LoadIcon (string name, Gtk.IconSize size, IconLookupFla
...
+		/// Loads the a image for a given name and size (icons must be squared)
 		/// </summary>
 		/// <returns>The icon as a Gdk.Image or missing image icon if not found.</returns>
 		/// <param name="name">Icon Name.</param>
-		/// <param name="width">Icon Width as a Gtk.IconSize.</param>
-		/// <param name="width">Icon Height as a Gtk.IconSize.</param>
+		/// <param name="size">Icon size as a int.</param>
 		/// <param name="flags">Lookup Flags like ForceSVG.</param>
-		public static Gtk.Image LoadImageIcon (string name, int width, int height, IconLookupFlags flags = IconLookupFlags.ForceSvg)
+		public static Gtk.Image LoadImageIcon (string name, int size, IconLookupFlags flags = IconLookupFlags.ForceSvg)
```
Comment (vmartos) [inline VAS.UI.Helpers.Gtk2/Misc.cs]: this method is not needed now, it makes the same but just returning an image, In fact we want an image to be returned, we can let this method but i will do:

`public static Gtk.Image LoadImageIcon (string name, int size, IconLookupFlags flags = IconLookupFlags.ForceSvg)
{
     return new Gtk.Image(Misc.LoadIcon (name, size));
}`

Comment (aameijide) [inline VAS.UI.Helpers.Gtk2/Misc.cs]: Yes, it seems logic, also, as we talked, is better to initialise directly than having a method.

**Comment by aameijide** [comment]
Discarding this pull request. The changes that it should made doesn't have sense now.

**Pull request closed without merging**